bin/apply.bash

```diff
@@ -31,14 +31,14 @@ ansible-playbook -i "${config[inventory_file]}" cluster.yml -b "${@}"
 log_info "Kubespray done"
 
 log_info "Get kubeconfig"
-ansible-playbook -i "${config[inventory_file]}" ../playbooks/kubeconfig.yml -b
+ansible-playbook -i "${config[inventory_file]}" ../playbooks/kubeconfig.yml -b "${@}"
 log_info "Adding cluster-admin ClusterRoleBinding"
-ansible-playbook -i "${config[inventory_file]}" ../playbooks/cluster_admin_rbac.yml -b
+ansible-playbook -i "${config[inventory_file]}" ../playbooks/cluster_admin_rbac.yml -b "${@}"
```

Comment:
Only issue I see with this is that this will not apply if you don't include the first control-plane node. I guess this is fine and I don't see any other solution around this.
Maybe a small warning message saying that this might not've worked as expected if this is set to anything other than `""`? What do you think about that?

Reply:
Do you mean for example if you are using the `--limit` flag which does not include the first control-plane node when running apply? We wanted to add this to be able to provide the `--user` flag mainly, the old behavior should still work, but I added a warning that is only shown if the user provides additional arguments to `apply`: https://github.com/elastisys/compliantkubernetes-kubespray/commit/a0e2ebcb49033379813cde5eecea7d0d32869e63